src/Core/Defines.h

```diff
@@ -27,7 +27,7 @@
   *  and if the parts were small, then it would be costly then to combine them.
   */
 #define DEFAULT_INSERT_BLOCK_SIZE \
-    1048449 /// 1048576 - PADDING_FOR_SIMD - (PADDING_FOR_SIMD - 1) bytes padding that we usually have in arrays
+    65409 /// 65536 - PADDING_FOR_SIMD - (PADDING_FOR_SIMD - 1) bytes padding that we usually have in arrays
```

Comment:
Decrease DEFAULT_INSERT_BLOCK_SIZE if there is not enough RAM.
this is for saving memory, but we can still change this by query level settings.

Reply:
Ingest block size is usually not tunable at query time